lib/persisters/local.js

```diff
@@ -9,41 +9,27 @@ const fs = require('fs')
  */
 
 class File {
-  constructor (opts) {
+  constructor(opts) {
     if (!opts.path) {
       throw new Error('`path` is undefined for local persister')
     }
 
-    const directory = path.resolve(__dirname, opts.path)
+    const directory = path.resolve(__dirname, opts.path.split('/').slice(0, -1).join('/'))
     mkdirp(directory)
 
-    this.directory = directory
+    this.directory = opts.path
```

Comment:
seems like you're assigning a file path to `directory` which is kind of confusing. why not `this.path`?